src/algebra/order/with_zero.lean

```diff
@@ -50,6 +50,100 @@ instance [linear_ordered_add_comm_group_with_top α] :
   ..multiplicative.linear_ordered_comm_monoid_with_zero,
   ..multiplicative.nontrivial }
 
+section monoid
+variable [monoid α]
+
+section preorder
+variable [preorder α]
+
+section left
+variable [covariant_class α α (*) (≤)]
+
+lemma left.one_le_pow_of_le : ∀ {n : ℕ} {x : α}, 1 ≤ x → 1 ≤ x^n
+| 0       x _ := (pow_zero x).symm.le
+| (n + 1) x H := calc 1 ≤ x          : H
+                    ... = x * 1      : (mul_one x).symm
+                    ... ≤ x * x ^ n  : mul_le_mul_left' (left.one_le_pow_of_le H) x
+                    ... = x ^ n.succ : (pow_succ x n).symm
+
+--alias left.one_le_pow_of_le ← one_le_pow_of_one_le'
```

Comment:
Is this intentionally commented out?

Reply:
The lemma name `one_le_pow_of_one_le'` is already used, so I simply removed the comment.  Now that the refactor is getting to lemmas that already exist, there might be a little bit of cleaning up to remove duplicates.

I would leave this for subsequent PRs, though!